WalletWasabi.Fluent/ViewModels/Wallets/Send/TransactionPreviewViewModel.cs

```diff
@@ -227,29 +227,29 @@ private async Task OnChangePocketsAsync()
 
 	private async Task<bool> InitialiseTransactionAsync()
```

Comment:
The diff looks really ugly here, but if I understand it all that happened is that you changed the order of the two `if` conditions. If so, why?

Reply:
Yes, it is just an order change. The reason is previously first the pocket selection happened and the fee rate selection was only after. In this PR the fee rate is needed during pocket selection so I changed the order. It does not affect anything.